Tools/CISupport/build-webkit-org/factories.py

```diff
@@ -115,7 +115,7 @@ def __init__(self, platform, configuration, architectures, additionalArguments=N
             self.addStep(ExtractTestResults())
             self.addStep(SetPermissions())
 
-        if platform.startswith('win') or platform.startswith('mac') or platform.startswith('ios-simulator'):
+        if platform.startswith(('win', 'mac', 'ios-simulator', 'visionos-simulator')):
```

Comment:
It might be worth removing this one for now, I don't know the state of API tests for visionOS simulator, and I'm not even sure that we've tried running them at all.

Reply:
I removed it in both places.